### PR TITLE
[Validator] Consistently use "This value is not" instead of "This is not" in error messages

### PR DESCRIPTION
**Pull request by nicolas-grekas**
| Q             | A
| ------------- | ---
| Branch?       | 5.4
| Bug fix?      | no
| New feature?  | no
| Deprecations? | no
| Issues        | -
| License       | MIT

That's longer but more understandable. That's also how most messages are constructed already.

Related to #53417

### CODE REVIEW

File: src/Symfony/Component/Validator/Resources/translations/validators.de.xlf [hunk 1]
```diff
@@ -312,15 +312,15 @@
             </trans-unit>
             <trans-unit id="81">
                 <source>This is not a valid Business Identifier Code (BIC).</source>
-                <target>Dieser Wert ist keine gültige internationale Bankleitzahl (BIC).</target>
+                <target state="needs-review-translation">Dieser Wert ist kein gültiger Geschäftsidentifikationscode (BIC).</target>
```
Comment (xabbuh) [inline src/Symfony/Component/Validator/Resources/translations/validators.de.xlf]:
```suggestion
                <target state="needs-review-translation">Dieser Wert ist keine gültige internationale Bankleitzahl (BIC).</target>
```

Comment (nicolas-grekas) [reply src/Symfony/Component/Validator/Resources/translations/validators.de.xlf]: updated thanks

File: src/Symfony/Component/Validator/Resources/translations/validators.fr.xlf [hunk 2]
```diff
@@ -136,7 +136,7 @@
             </trans-unit>
             <trans-unit id="37">
                 <source>This is not a valid IP address.</source>
-                <target>Cette adresse IP n'est pas valide.</target>
+                <target state="needs-review-translation">Cette valeur n'est pas une adresse IP valide.</target>
```
Comment (GromNaN) [inline src/Symfony/Component/Validator/Resources/translations/validators.fr.xlf]: This is correct 🥂

Comment (nicolas-grekas) [reply src/Symfony/Component/Validator/Resources/translations/validators.fr.xlf]: attribute removed :)

Comment (GromNaN) [reply src/Symfony/Component/Validator/Resources/translations/validators.fr.xlf]: Others, but not this one.

Comment (nicolas-grekas) [reply src/Symfony/Component/Validator/Resources/translations/validators.fr.xlf]: :facepalm:

File: src/Symfony/Component/Validator/Resources/translations/validators.hr.xlf [hunk 3]
```diff
@@ -224,7 +224,7 @@
             </trans-unit>
             <trans-unit id="59">
                 <source>This is not a valid International Bank Account Number (IBAN).</source>
-                <target>Ova vrijednost nije ispravan međunarodni broj bankovnog računa (IBAN).</target>
+                <target state="needs-review-translation">Ova vrijednost nije valjani Međunarodni bankovni broj računa (IBAN).</target>
```
Comment (HypeMC) [inline src/Symfony/Component/Validator/Resources/translations/validators.hr.xlf]: No need for the capital M:
```suggestion
                <target state="needs-review-translation">Ova vrijednost nije valjani međunarodni bankovni broj računa (IBAN).</target>
```

File: src/Symfony/Component/Validator/Resources/translations/validators.hr.xlf [hunk 4]
```diff
@@ -312,15 +312,15 @@
             </trans-unit>
             <trans-unit id="81">
                 <source>This is not a valid Business Identifier Code (BIC).</source>
-                <target>Ovo nije validan poslovni identifikacijski broj (BIC).</target>
+                <target state="needs-review-translation">Ova vrijednost nije valjani Poslovni identifikacijski kod (BIC).</target>
```
Comment (HypeMC) [inline src/Symfony/Component/Validator/Resources/translations/validators.hr.xlf]: Same here:
```suggestion
                <target state="needs-review-translation">Ova vrijednost nije valjani poslovni identifikacijski kod (BIC).</target>
```

**Review by asispts — APPROVED** [review]
`state="needs-review-translation"` can be removed from Indonesian (id) file.

File: src/Symfony/Component/Validator/Resources/translations/validators.de.xlf (hunk 1 above)
Comment (nicolas-grekas) [reply src/Symfony/Component/Validator/Resources/translations/validators.de.xlf]: I assumed other translation are correct in "de" and removed the attribute

File: src/Symfony/Component/Validator/Resources/translations/validators.hr.xlf (hunk 4 above)
Comment (nicolas-grekas) [reply src/Symfony/Component/Validator/Resources/translations/validators.hr.xlf]: I assumed other translations are correct in "hr" and removed the attribute

File: src/Symfony/Component/Validator/Resources/translations/validators.nl.xlf [hunk 5]
```diff
@@ -136,7 +136,7 @@
             </trans-unit>
             <trans-unit id="37">
                 <source>This is not a valid IP address.</source>
-                <target>Dit is geen geldig IP-adres.</target>
+                <target state="needs-review-translation">Deze waarde is geen geldig IP-adres.</target>
```
Comment (rosier) [inline src/Symfony/Component/Validator/Resources/translations/validators.nl.xlf]: Correct

File: src/Symfony/Component/Validator/Resources/translations/validators.nl.xlf [hunk 6]
```diff
@@ -224,7 +224,7 @@
             </trans-unit>
             <trans-unit id="59">
                 <source>This is not a valid International Bank Account Number (IBAN).</source>
-                <target>Dit is geen geldig internationaal bankrekeningnummer (IBAN).</target>
+                <target state="needs-review-translation">Deze waarde is geen geldig Internationaal Bankrekeningnummer (IBAN).</target>
```
Comment (rosier) [inline src/Symfony/Component/Validator/Resources/translations/validators.nl.xlf]:
```suggestion
                <target>Deze waarde is geen geldig internationaal bankrekeningnummer (IBAN).</target>
```

File: src/Symfony/Component/Validator/Resources/translations/validators.nl.xlf [hunk 7]
```diff
@@ -312,15 +312,15 @@
             </trans-unit>
             <trans-unit id="81">
                 <source>This is not a valid Business Identifier Code (BIC).</source>
-                <target>Dit is geen geldige bedrijfsidentificatiecode (BIC/SWIFT).</target>
+                <target state="needs-review-translation">Deze waarde is geen geldige Zakelijke Identificatie Code (BIC).</target>
```
Comment (rosier) [inline src/Symfony/Component/Validator/Resources/translations/validators.nl.xlf]:
```suggestion
                <target>Deze waarde is geen geldige zakelijke identificatiecode (BIC).</target>
```

File: src/Symfony/Component/Validator/Resources/translations/validators.nl.xlf [hunk 8]
```diff
@@ -312,15 +312,15 @@
...
             </trans-unit>
             <trans-unit id="82">
                 <source>Error</source>
                 <target>Fout</target>
             </trans-unit>
             <trans-unit id="83">
                 <source>This is not a valid UUID.</source>
-                <target>Dit is geen geldige UUID.</target>
+                <target state="needs-review-translation">Deze waarde is geen geldige UUID.</target>
```
Comment (rosier) [inline src/Symfony/Component/Validator/Resources/translations/validators.nl.xlf]: Correct

File: src/Symfony/Component/Validator/Resources/translations/validators.nl.xlf [hunk 9]
```diff
@@ -436,7 +436,7 @@
             </trans-unit>
             <trans-unit id="112">
                 <source>This is not a valid MAC address.</source>
-                <target state="needs-translation">This is not a valid MAC address.</target>
+                <target state="needs-review-translation">Deze waarde is geen geldig MAC-adres.</target>
```
Comment (rosier) [inline src/Symfony/Component/Validator/Resources/translations/validators.nl.xlf]: Correct

**Review by rosier — CHANGES_REQUESTED** [review]

**Comment by nicolas-grekas** [comment]
PR rebased

File: src/Symfony/Component/Validator/Resources/translations/validators.pl.xlf [hunk 10]
```diff
@@ -312,15 +312,15 @@
             </trans-unit>
             <trans-unit id="81">
                 <source>This is not a valid Business Identifier Code (BIC).</source>
-                <target>Ta wartość nie jest poprawnym kodem BIC (Business Identifier Code).</target>
+                <target state="needs-review-translation">Ta wartość nie jest prawidłowym Kodem Identyfikującym Biznes (BIC).</target>
```
Comment (michaljusiega) [inline src/Symfony/Component/Validator/Resources/translations/validators.pl.xlf]:
```suggestion
                <target>Ta wartość nie jest prawidłowym Kodem Identyfikującym Bank (BIC).</target>
```

I think it should be like this.

Comment (michaljusiega) [reply src/Symfony/Component/Validator/Resources/translations/validators.pl.xlf]: Other changes in `pl.xlf` LGTM.

Comment (nicolas-grekas) [reply src/Symfony/Component/Validator/Resources/translations/validators.pl.xlf]: updated, thanks for the review

**Comment by xabbuh** [comment]
@nicolas-grekas After #53444 we now need to update the source of `112` everywhere to "This value is not a valid MAC address.".

**Comment by nicolas-grekas** [comment]
> This value is not a valid MAC address.

updated!

**Pull request merged**